projects/robots/rec/fabtino/protos/Fabtino.proto

```diff
@@ -30,29 +30,34 @@ PROTO Fabtino [
     children [
       Group {
         children [
-          Shape {
-            appearance DEF LIGHT_GRAY PBRAppearance {
-              baseColor 0.78 0.78 0.78
-              metalness 0.3
-              roughness 0.1
-            }
-            geometry Mesh {
-              url [
-                "webots://projects/robots/rec/fabtino/protos/meshes/base_central.dae"
-              ]
-            }
-          }
-          Shape {
-            appearance DEF DARK_GRAY PBRAppearance {
-              baseColor 0.43 0.43 0.43
-              metalness 0.3
-              roughness 0.1
-            }
-            geometry Mesh {
-              url [
-                "webots://projects/robots/rec/fabtino/protos/meshes/base.dae"
-              ]
-            }
+          Transform {
+            rotation 0.57735 0.57735 0.57735 2.09439
```

Comment:
I would rather change the upper Group node to a Transform instead of adding an extra layer.